.ahoy.yml

```diff
@@ -51,51 +51,51 @@ commands:
 
   cli:
     usage: Start a shell inside TEST container.
-    cmd: docker-compose exec cli bash
+    cmd: docker compose exec cli bash
 
   run:
     usage: Run command inside TEST container.
-    cmd: docker-compose exec -T cli bash -c "$@"
+    cmd: docker compose exec -T cli bash -c "$@"
 
   drush:
     usage: Run drush commands in TEST container.
-    cmd: docker-compose exec -T cli drush "$@"
+    cmd: docker compose exec -T cli drush "$@"
 
   logs:
     usage: Show Docker logs.
-    cmd: docker-compose logs "$@"
+    cmd: docker compose logs "$@"
 
   ps:
     usage: List running Docker containers.
-    cmd: docker-compose ps
+    cmd: docker compose ps
 
   restart:
     usage: Restart Docker containers.
-    cmd: docker-compose restart
+    cmd: docker compose restart
 
   stop:
     usage: Stop Docker containers.
-    cmd: docker-compose stop "$@"
+    cmd: docker compose stop "$@"
 
   install:
     usage: Install the profile.
-    cmd: docker-compose exec -T cli drush si -y govcms "$@"
+    cmd: docker compose exec -T cli drush si -y govcms "$@"
 
   login:
     usage: Login to a website.
-    cmd: docker-compose exec -T cli drush uli "$@"
+    cmd: docker compose exec -T cli drush uli "$@"
 
   pull:
     usage: Pull latest docker images.
     cmd: docker image ls --format \"{{.Repository}}:{{.Tag}}\" | grep govcms/ | grep -v none | xargs -n1 docker pull | cat
 
   audit-site:
     usage: Run site audit
```

Comment:
I believe drutiny does not exist anymore in the image. So this could be removed?
```suggestion
```